### ISSUE TITLE
`annotation-no-unknown` false-positive

### ISSUE THREAD

**Issue by boris-petrov** [stylelint-scss/stylelint-config-recommended-scss#149]
I have a variable definition:

```scss
$postfix: if($infix != '', $infix + '-down', '');
```

And the `!=` leads to a `annotation-no-unknown` warning which is wrong.

**Comment by XhmikosR** [comment]
@kristerkari it seems there are more false positives apart from `!=`: https://github.com/twbs/stylelint-config-twbs-bootstrap/actions/runs/3368636492/jobs/5587403137

**Comment by XhmikosR** [comment]
We can work around these, but makes me wonder if it's already too strict maybe? We can't be the only ones using such code.

**Comment by XhmikosR** [comment]
Yeah, I'd just disable the rule then.

**Comment by katzmo** [comment]
The rule is still present in `stylelint-config-recommended-scss-9.0.0`, with only "default" and "global" ignored.

**Comment by kristerkari** [comment]
Hmm.. yeah it seems that I forgot to do it.

**Comment by kristerkari** [comment]
Fix here:
https://github.com/stylelint-scss/stylelint-config-recommended-scss/pull/186

**Comment by kristerkari** [comment]
Fix available in the newest version:
https://github.com/stylelint-scss/stylelint-config-recommended-scss/releases/tag/v9.0.1

**Comment by katzmo** [comment]
Wow, that’s a quick reaction! Thank you @kristerkari!
